src/client/modals/source-modal/source-modal.tsx

```diff
@@ -57,9 +55,11 @@ export class SourceModal extends React.Component<SourceModalProps, SourceModalSt
     >
       <div className="content">
         {header}
-        <SyntaxHighlighter className="source-modal__source" language="json" style={githubGist}>
-          {source}
-        </SyntaxHighlighter>
+        <React.Suspense fallback={Loader}>
```

Comment:
Perfect case for lazy component!

Reply:
Yeah, it is! Wonder if `SourceModal` wouldn't be a better candidate though.